app/controllers/comments_controller.rb

```diff
@@ -1,21 +1,22 @@
 class CommentsController < ApplicationController
+  before_action :get_article, only: [:create, :destroy]
+
   def create
-    @article = Article.find(params[:article_id])
-    @comment = @article.comments.create(comment_params)
+    @user = current_user
+    @comment = @article.comments.create({message: params[:comment][:message], user_id: @user.id})
 
     redirect_to article_path(@article)
   end
 
   def destroy
-    @article = Article.find(params[:article_id])
     @comment = @article.comments.find(params[:id])
 
     @comment.destroy
     redirect_to article_path(@article)
   end
 
   private
-    def comment_params
```

Comment:
I removed this function because I am manually sending only params[:comment][:message] field to create function